src/blockdom/markup.ts

```diff
@@ -0,0 +1,31 @@
+import { html } from "./html";
+import { toggler } from "./toggler";
+import { text } from "./text";
+
+class Markup extends String {}
+
+export function markup(value: any) {
+  return new Markup(value);
+}
+
+type Safe = ReturnType<typeof toggler>;
+export function safeOut(value: any): Safe {
+  if (!value) {
+    return value
+  }
+  let safeKey;
+  let node;
+  if (value instanceof Markup) {
+    safeKey = `string_safe`;
+    node = html(value as string);;
+  } else if (typeof value === "string") {
+    safeKey = "string_unsafe";
+    node = text(value);
+  } else if (value.constructor.name === "Block") {
```

Comment:
this is actually false. it could be any kind of block, not just a Block. what about removing the `if` and assuming that it is a block?